python/ray/includes/metric.pxd

```diff
@@ -0,0 +1,45 @@
+from libcpp.string cimport string as c_string
+from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector as c_vector
+
+cdef extern from "opencensus/tags/tag_key.h" nogil:
+    cdef cppclass CTagKey "opencensus::tags::TagKey":
+        @staticmethod
+        CTagKey Register(c_string &name)
+        const c_string &name() const
+
```

Comment:
You need 2 new lines here

Reply:
> You need 2 new lines here

Got it. But I noticed that it's only 1 blank line between two class definition in libcoreworker.pxd. So I'll keep them in *.pxd and add addtional blank line after class definition in *.pxi.

Reply:
Hmm that's right.. I think it is not consistent lol. If you look at function_descriptor.pxd, it has two blank lines.